magento/pwa-studio
# PWA-1707: Cleanup whishlist for render empty

## Description

Clean up single wishlist

## Related Issue

Closes PWA-1707

## Acceptance





### Verification Stakeholders



### Specification




### Verification Steps




My Account > Favorites shows a "there are currently no items in this list" message
After items are added and then removed back down to zero items, the "there are currently no items in this list" message shows again
Test Plan

Navigate to My Account > Favorites with a single, empty wishlist
See that the page shows the "no items" message
Add an item to the wishlist
Remove the item from the wishlist
See that the page again shows the "no items" message

## Screenshots / Screen Captures (if appropriate)
![Screenshot from 2021-06-10 14-05-44](https://user-images.githubusercontent.com/5289370/121522000-f71ce000-c9f4-11eb-9a31-986fc1077b11.png)


## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
